workflows/real_time_ai_surgical_video_processing/README.md

```diff
@@ -0,0 +1,152 @@
+# Real-Time AI Surgical Video Processing Workflow
+
+## Overview
+
+In this workflow, we demonstrate a comprehensive real-time AI surgical video processing workflow that includes:
+
+1. Out-of-body detection to determine if the endoscope is inside or outside the patient
+2. Dynamic flow condition based on out-of-body detection results:
+   - Deidentification (pixelation) when outside the body
+   - Multi-AI processing when inside the body
+3. Multi-AI processing with:
+   - SSD detection for surgical tool detection
+   - MONAI segmentation for endoscopic tool segmentation
+
+![Sample Endoscopy Image](images/RAISVP-sample-image.png)
+
+Fig. 1 Endoscopy (laparoscopy) image from a cholecystectomy (gallbladder removal surgery) showing tool detection and segmentation results from two concurrently executed AI models.
+Image courtesy of Research Group Camma, IHU Strasbourg and the University of Strasbourg ([NGC Resource](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_endoscopy_sample_data))
+
+Please refer to the README under [./app_dev_process](./app_dev_process/README.md) to see the process of developing the applications.
+
+The application graph looks like:
+![RAISVP-workflow](./images/RAISVP-dynamic-workflow.png)
+
+## Models
+
+This workflow utilizes three AI models:
+
+- [Out-of-body detection model](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/endoscopy_out_of_body_detection): `out_of_body_detection.onnx`
+- [SSD model from NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/ssd_surgical_tool_detection_model) with additional NMS op: `epoch24_nms.onnx`
+- [MONAI tool segmentation model from NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/monai_endoscopic_tool_segmentation_model): `model_endoscopic_tool_seg_sanitized_nhwc_in_nchw_out.onnx`
+
+## Data
+
+[📦️ (NGC) Sample App Data for AI-based Endoscopy Tool Tracking](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_endoscopy_sample_data)
+
+[📦️ (NGC) Sample App Data for Out-of-Body Detection](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_endoscopy_sample_data)
+
+## Requirements
```

Comment:
Let's start the README with a quick start using the `build_and_run` command.
I would limit the workflow to build_and_run for maintenance purposes.

Reply:
Thanks for this feedback! I made a "Quick start guide" section that uses only `build_and_run`, and moved the rest to "Advanced usage" section